src/awfy.js

```diff
@@ -360,14 +360,14 @@ const MOBILE_APPS = {
     name: 'fenix',
     label: 'Fenix-nofis',
     color: PALETTE.orange,
-    project: 'firefox-android',
+    project: ALT_PROJECT,
     extraOptions: ['webrender'],
   },
   'fenix-fission': {
     name: 'fenix',
     label: 'Fenix-fission',
     color: PALETTE.emerald,
-    project: 'firefox-android',
+    project: PROJECT,
```

Comment:
Nice, glad this is getting fixed! :) What do you think about using autoland or `ALT_PROJECT` instead? It'll match what we display for geckoview that way: https://github.com/mozilla-frontend-infra/firefox-performance-dashboards/blob/e4bf94523cb128547e1fd00a13cbb43f07a1c59b/src/awfy.js#L377

Reply:
Oh nvm! I see you've got it right already, I just realized I was looking at the fission variant.